pr_agent/tools/pr_line_questions.py

```diff
@@ -92,6 +100,73 @@ async def run(self):
                 self.git_provider.publish_comment(model_answer_sanitized)
 
         return ""
+        
+    def _load_conversation_history(self):
+        """generate conversation history from the code review thread"""
+        try:
+            comment_id = get_settings().get('comment_id', '')
+            file_path = get_settings().get('file_name', '')
+            line_number = get_settings().get('line_end', '')
+            
+            # return if no comment id or file path and line number
+            if not (comment_id or (file_path and line_number)):
+                return
```

Comment:
Perfect! Very important to stop for an issue to save time and resources.
Practically, we need to have all objects not empty, right?
```suggestion
            # return if any required parameter is missing
            if not all([comment_id, file_path, line_number]):
                return
```